tests/xdsl_tblgen/test.td

```diff
@@ -0,0 +1,87 @@
+// Tablegen file used to generate `test.json`, which can be generated with the following command:
+// ```
+// llvm-tblgen -D test test.td -I $LLVM/mlir/include --dump-json | xdsl-tblgen -c | jq > test.json
```

Comment:
?
```suggestion
// llvm-tblgen -D test test.td -I $LLVM/mlir/include --dump-json | xdsl-tblgen -c | jq --sort-keys > test.json
```

Reply:
I feel this fixes the symptom rather than the cause, so will investigate what is making `xdsl-tblgen -c` non-deterministic